lib/aruba/api.rb

```diff
@@ -729,6 +735,61 @@ def io_wait
       @aruba_io_wait_seconds || DEFAULT_IO_WAIT_SECONDS
     end
 
+    DEFAULT_ROOT_DIRECTORY = Dir.getwd
+
+    # The root directory of aruba
+    def root_directory
+      @aruba__root_directory ||= DEFAULT_ROOT_DIRECTORY
+    end
+
+    DEFAULT_FIXTURES_PATH_PREFIX = '%/'
+
+    def fixtures_path_prefix
+      @aruba_fixtures_path_prefix || DEFAULT_FIXTURES_PATH_PREFIX
+    end
+
+    DEFAULT_FIXTURES_DIRECTORY = 'fixtures'
+
+    # The path to the directory which contains fixtures
+    # You might want to overwrite this method to place your data else where.
+    #
+    # @return [String]
+    #   The directory to where your fixtures are stored
+    def fixtures_directory
+      if_not_found = proc { File.join(root_directory, DEFAULT_FIXTURES_DIRECTORY) }
+      %w(
+        fixtures
+        spec/fixtures
+        features/fixtures
+        test/fixtures
+      ).map { |dir| File.join(root_directory, dir) }.find(if_not_found) { |dir| File.directory? dir }
+    end
+
+    # Return known fixtures
+    #
+    # @return [Array]
+    #   An array of known files and directories in fixtures directory
+    def known_fixtures
```

Comment:
I consider this to be feature creep. Stick to the simplest possible solution first. Then open another PR for this feature and argue why this is necessary in Aruba.